src/VisualStudio/CSharp/Impl/LanguageService/CSharpCodeCleanupFixer.cs

```diff
@@ -0,0 +1,348 @@
+        public static readonly FixIdDefinition UseExpressionBodyForAccessorsDiagnosticId;
+
+        [Export]
+        [FixId(IDEDiagnosticIds.InlineDeclarationDiagnosticId)]
+        [Name(IDEDiagnosticIds.InlineDeclarationDiagnosticId)]
+        [Order(After = IDEDiagnosticIds.UseImplicitTypeDiagnosticId)]
+        [ConfigurationKey("unused")]
+        [HelpLink("https://www.microsoft.com")]
+        [LocalizedName(typeof(CSharpFeaturesResources), nameof(CSharpFeaturesResources.Apply_inline_out_variable_preferences))]
+        public static readonly FixIdDefinition InlineDeclarationDiagnosticId;
+
+        [Export]
+        [FixId(CSharpRemoveUnusedVariableCodeFixProvider.CS0168)]
+        [Name(CSharpRemoveUnusedVariableCodeFixProvider.CS0168)]
+        [Order(After = IDEDiagnosticIds.RemoveUnnecessaryCastDiagnosticId)]
+        [ConfigurationKey("unused")]
+        [HelpLink("https://www.microsoft.com")]
+        [LocalizedName(typeof(CSharpFeaturesResources), nameof(CSharpFeaturesResources.Remove_unused_variables))]
+        public static readonly FixIdDefinition CS0168;
+
+        [Export]
+        [FixId(CSharpRemoveUnusedVariableCodeFixProvider.CS0219)]
+        [Name(CSharpRemoveUnusedVariableCodeFixProvider.CS0219)]
+        [Order(After = IDEDiagnosticIds.RemoveUnnecessaryCastDiagnosticId)]
+        [ConfigurationKey("unused")]
+        [HelpLink("https://www.microsoft.com")]
+        [LocalizedName(typeof(CSharpFeaturesResources), nameof(CSharpFeaturesResources.Remove_unused_variables))]
+        public static readonly FixIdDefinition CS0219;
+
+        [Export]
+        [FixId(IDEDiagnosticIds.UseObjectInitializerDiagnosticId)]
+        [Name(IDEDiagnosticIds.UseObjectInitializerDiagnosticId)]
+        [Order(After = IDEDiagnosticIds.PreferBuiltInOrFrameworkTypeDiagnosticId)]
+        [ConfigurationKey("unused")]
+        [HelpLink("https://www.microsoft.com")]
+        [LocalizedName(typeof(CSharpFeaturesResources), nameof(CSharpFeaturesResources.Apply_object_collection_initialization_preferences))]
+        public static readonly FixIdDefinition UseObjectInitializerDiagnosticId;
+
+        [Export]
+        [FixId(IDEDiagnosticIds.UseCollectionInitializerDiagnosticId)]
+        [Name(IDEDiagnosticIds.UseCollectionInitializerDiagnosticId)]
+        [Order(After = IDEDiagnosticIds.PreferBuiltInOrFrameworkTypeDiagnosticId)]
+        [ConfigurationKey("unused")]
+        [HelpLink("https://www.microsoft.com")]
+        [LocalizedName(typeof(CSharpFeaturesResources), nameof(CSharpFeaturesResources.Apply_object_collection_initialization_preferences))]
+        public static readonly FixIdDefinition UseCollectionInitializerDiagnosticId;
+
+        [Export]
+        [FixId(RemoveUnusedImportsFixId)]
+        [Name(RemoveUnusedImportsFixId)]
+        [ConfigurationKey("unused")]
+        [HelpLink("https://www.microsoft.com")]
+        [ExportMetadata("EnableByDefault", true)]
```

Comment:
The editor uses this to enable a fixer initially. ExportMetadata instead of a proper attribute because there is none yet and we have no time for it in Preview1. It's going to be quite different post-Preview1 anyway I suppose.

Reply:
Consolidating discussion of this to https://github.com/dotnet/roslyn/pull/30039#discussion_r226546564